test/governance/extensions/GovernorTimelockCompound.test.js

```diff
@@ -328,6 +328,57 @@ contract('GovernorTimelockCompound', function (accounts) {
     runGovernorWorkflow();
   });
 
+  describe('relay', function () {
```

Comment:
Does it make sense to add a test for plain Governor without a timelock?

Reply:
The Governor without a timelock already acts by itself. using relay would add an indirection level and not add anything. So it doesn't add anything to the non-timelock flavors ... but we can still test it (and test its protected)